web/src/pages/Vulnerability/VulnerabilityPage.jsx

```diff
@@ -52,14 +73,23 @@ export function Vulnerability() {
   const hrefToServicePage = `${rootPrefix}/?` + pteamServiceParams.toString();
   const hrefToPackagePage = `${rootPrefix}/packages/${packageId}?` + pteamServiceParams.toString();
 
+  const currentPackageDependencies = (serviceDependencies ?? []).filter(
+    (dependency) => dependency.package_id === packageId,
+  );
+  const references = currentPackageDependencies.map((dependency) => ({
+    package_name: dependency.package_name,
+    package_source_name: dependency.package_source_name,
+    ecosystem: dependency.package_ecosystem,
+  }));
```

Comment:
- 変数名がreferences のままだと、VulnTableRowView.jsxのreferences と同じデータだと誤認されるので、変えるべき
- 複数件要らない

```suggestion
  const currentPackage = {
    package_name: currentPackageDependencies[0].package_name,
    package_source_name: currentPackageDependencies[0].package_source_name,
    ecosystem: currentPackageDependencies[0].package_ecosystem,
  };
```
これと同じcurrentPackage データをVulnTableRowViewでも作ってVulnerabilityDrawerに渡せば良いと思います。

Reply:
vulnerable_packagesとcurrentPackage を受け取って matchedVulnPackage を返す、という処理が３か所あるので、関数に切り出して共通化して欲しいです。
処理するのはAffectのデータなのでvulnUtils.jsに定義して良いと思います。